docs/source/glossary.rst

```diff
@@ -42,22 +48,30 @@ defines that the access to the life cycle chaincode ``GetDeploymentSpec`` API
 (the resource) is accessible by identities which satisfy the
 ``/Channel/Application/Readers`` policy.
 
+ACL被格式化为键值对列表，其中键标识我们希望控制其访问的资源，其值标识允许访问它的通道策略（组）。 例如， ``lscc/GetDeploymentSpec: /Channel/Application/Readers`` 定义对生命周期链代码 ``GetDeploymentSpec`` API（资源）的访问可由满足 ``/Channel/Application/Readers`` 策略的标识访问。
+
 A set of default ACLs is provided in the ``configtx.yaml`` file which is
 used by configtxgen to build channel configurations. The defaults can be set
 in the top level "Application" section of ``configtx.yaml`` or overridden
 on a per profile basis in the "Profiles" section.
 
+``configtx.yaml`` 文件中提供了一组默认ACL，configtxgen使用该文件来构建通道配置。可以在 ``configtx.yaml`` 的顶级“应用程序”部分中设置默认值，也可以在“配置文件”部分中按每个配置文件覆盖默认值。
+
 .. _Block:
 
-Block
+Block - 区块
 -----
 
 An ordered set of transactions that is cryptographically linked to the
 preceding block(s) on a channel.
 
+区块是通道上一组有序交易的集合，通过密码学手段（哈希加密）连接到前导区块。
+
+Zhu Jiang：区块是一组有序的交易集合，在通道中经过加密（哈希加密）后与前序区块连接。
```

Comment:
I'm fine by copying from the wiki page. But we may need to handle these comments.

Reply:
Refer to Fabric 1.1 cn, I kept the explanations of Zhu Jiang, so maybe it would be better to delete it?

Reply:
> I'm fine by copying from the wiki page. But we may need to handle these comments.

Refer to Fabric 1.1 cn, I kept the explanations of Zhu Jiang, so maybe it would be better to delete it?

Reply:
“区块是一组有序的交易集合，在通道中经过加密（哈希加密）后与前序区块连接。” 留着这一句就可以了。

Reply:
> “区块是一组有序的交易集合，在通道中经过加密（哈希加密）后与前序区块连接。” 留着这一句就可以了。

已经改好啦~